Documentation/Updating.md

```diff
@@ -1,9 +1,14 @@
 # Updating the Microsoft Mixed Reality Toolkit
 
+- [How to update](#How-to-update-your-project-to-use-a-new-version-of-MRTK)
 - [2.1.0 to 2.2.0](#updating-210-to-220)
 - [2.0.0 to 2.1.0](#updating-200-to-210)
 - [RC2 to 2.0.0](#updating-rc2-to-200)
 
+## How to update your project to use a new version of MRTK
```

Comment:
I would suggest removing the hyperlink above and the section header. Instead add a section so it stands out more and simplifies need of hyperlink/section
```
> [!NOTE]
> Instructions to properly migrate to the latest version of the Mixed Reality Toolkit are documented in the [release notes](ReleaseNotes.md) for each version. 
```

Reply:
It looks like the link was broken because it was too long - my shortened link passed validation.  But I'll remove it as you suggest, and replace with this note.